toolchain/check/testdata/basics/multifile.carbon

```diff
@@ -14,15 +14,17 @@ package B api;
 
 fn B() {}
 
-// CHECK:STDOUT: file "a.carbon" {
+// CHECK:STDOUT: --- a.carbon
+// CHECK:STDOUT: file {
 // CHECK:STDOUT:   %A: <function> = fn_decl @A
 // CHECK:STDOUT: }
 // CHECK:STDOUT:
 // CHECK:STDOUT: fn @A() {
 // CHECK:STDOUT: !entry:
 // CHECK:STDOUT:   return
 // CHECK:STDOUT: }
-// CHECK:STDOUT: file "b.carbon" {
+// CHECK:STDOUT: --- b.carbon
```

Comment:
Could we add a blank line below the file marker, and at the end of the file (or above the file marker) when there are multiple files? Just want it to stand apart from the other top-level things.

Reply:
Added newlines.

I'm not making it conditional because that would require awareness on the part of the driver of how output is constructed, which I'd like to avoid if possible.